decidim-admin/app/controllers/decidim/admin/components_controller.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+require_dependency "decidim/admin/application_controller"
+
+module Decidim
+  module Admin
+    # Controller that allows managing all the Admins.
+    #
+    class ComponentsController < ApplicationController
+      include Concerns::ParticipatoryProcessAdmin
+
+      helper_method :manifest
+
+      def new
+        authorize! :create, Component
+
+        component = Component.new(
+          participatory_process: participatory_process
+        )
+
+        @form = form(ComponentForm).from_model(component).tap do |form|
+          form.name = default_name(manifest)
```

Comment:
Unfortunately can't do that because default attributes are weird when using `translatable_attribute`. I feel it's a hack and should be reworked sometime in the future.